code/src/components/product-card/ProductCard.scss

```diff
@@ -38,4 +45,23 @@
     @include transition((all, "medium"));
     backdrop-filter: blur(spacing(2));
   }
+
+  &__footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: spacing(2);
+    border-top: 1px solid var(--spa-products-container-border-color, $light);
+  }
+
+  &__button {
+    border: 1px solid var(--spa-best-sellers-price-color, $black);
+    background-color: var(--spa-button-bg-color, $transparent);
+    color: var(--spa-best-sellers-price-color, $black);
+    gap: spacing(1);
+
+    &:hover {
+      background-color: var(--spa-hover-button-background-color, $light);
+    }
+  }
```

Comment:
We already have these styles included in our AppButton. Please, just use variant and size prop. These styles are redundant 
![image](https://github.com/idx-academy/spa-orders/assets/118522791/b959ff1c-28e4-4006-b5b7-be0ff8ab80af)

Reply:
remove `border` and set `variant`

Reply:
> remove `border` and set `variant`



> removed `border` and set `variant`